### PR TITLE
[CI] System report: osx-xcode-13.4.x

### PR DESCRIPTION
**Pull request by bitrise-infrabot**

### CODE REVIEW

File: system_reports/MACOS/M1/osx-xcode-13.4.x.log [hunk 1]
```diff
@@ -7,29 +7,26 @@
...
 * Python: Python 2.7.18
-* Node.js: v16.16.0
-* NPM: 8.11.0
+* Node.js: v16.17.0
+* NPM: 8.15.0
 * Yarn: 1.22.19
 * Java:   system
   1.8
   1.8.0.292
-  11
```
Comment (lpusok) [inline system_reports/MACOS/M1/osx-xcode-13.4.x.log:23]: Is it intended to be removed? @bitrise-tom

File: system_reports/MACOS/M1/osx-xcode-13.4.x.log [hunk 2]
```diff
@@ -2973,8 +2971,8 @@ android-32
 
  * SDK packages:
 * Command line Tools version: 6.0
-* Platform Tools version: 33.0.1
-* Emulator version: 31.2.10
```
Comment (lpusok) [inline system_reports/MACOS/M1/osx-xcode-13.4.x.log:2977]: Can we keep 31.2.10 on Intel? On M1 it does not matter.

File: system_reports/MACOS/M1/osx-xcode-13.4.x.log (hunk 1 above)
Comment (bitrise-tom) [reply system_reports/MACOS/M1/osx-xcode-13.4.x.log:23]: No, it should still be there. We will need to check the machine and see why it's gone.

Comment (lpusok) [reply system_reports/MACOS/M1/osx-xcode-13.4.x.log:23]: It is a jenv bug that causes it, but we somehow worked around it earlier. The sed included in macOS works different than in linux (which jenv expects)

Comment (bitrise-tom) [reply system_reports/MACOS/M1/osx-xcode-13.4.x.log:23]: image

So we definitely need this?

File: system_reports/MACOS/M1/osx-xcode-13.4.x.log (hunk 2 above)
Comment (fehersanyi-bitrise) [reply system_reports/MACOS/M1/osx-xcode-13.4.x.log:2977]: we son't hardcode the version just run `sdkmanager install emulator` so if it's like this then so be it

**Pull request merged**